README.md

```diff
@@ -2,28 +2,28 @@
 
 Big Sur macOS 11 Mac Pro patcher.
 
-Please read the Requirements section. Thank you!
+Before you start please read the [Requirements](#requirements) section. Thank you!
 
 ### The 5 Most Common Issues
-1. User did not disable SIP using El Capitan macOS 10.11 or newer Recovery OS to Execut this simple command `csrutil disable`
-2. ...did not set boot arguments to at least `nvram boot-args"-no_compat_check"`
-3. (Mac Pro / Xserve Early 2008 only) has an incompatible Bluetooth 2.0 (BT2.0+EDR Card) that causes a kernal panic. Future versions will disable the card via sofware and only for Big Sur. 802.11ac / BT4 Combo cards and adapters are cheap, plus you'll get AirDrop and Handoff support.
-4. ...did not read this entire readme file. I try to cover every possible angle and I update this page when new issues are discovered. 
-5. ...did not read existing open issues before posting a new issue. I keep a good portion open that are the most common issues found and solved. Issues have a way of repeating themselves. If it's a repeat, more than likely you'll find the correct answer there.
+1. User did not disable SIP using El Capitan macOS 10.11 or newer Recovery OS to execute this simple command: `csrutil disable`
+2. ... did not set boot arguments to at least: `nvram boot-args"-no_compat_check"`
+3. (Early 2008 Mac Pro / Early 2008 Xserve only) These machines has an incompatible Bluetooth 2.0 (BT2.0+EDR Card) that causes a kernal panic. Future versions will disable the card via sofware and only for Big Sur. 802.11ac / BT4 Combo cards and adapters are cheap, plus you'll get AirDrop and Handoff support.
+4. ... did not read this entire readme file. I try to cover every possible angle and I update this page when new issues are discovered. 
+5. ... did not read existing open issues before posting a new issue. I keep a good portion open that are the most common issues found and solved. Issues have a way of repeating themselves. If it's a repeat, more than likely you'll find the correct answer there.
 
-# 🍔 bigmac1.1
+## 🍔 bigmac1.1
 
-<img src="https://swift4me.com/images/splash/bigmac1f.png" width="100%">
+<img src="https://swift4me.com/images/splash/bigmac1f.png" width="45%">
 
 * Includes all latest patches like USB1.1 device support, Airdrop and more!
 * Added a new Startup Disk tool to the repo. Created by StarPlayrX.
 * Packaged in our classic v1.0 installer format
 * A Big Mac is better than a New Mac. Please donate to [NiceMac LLC](https://www.paypal.com/donate?hosted_button_id=M3U48FLF87SXQ)
 
 
-# 🍔🍦 bigmac2.0 
+## 🍔 bigmac2.0 is coming soon
 
-<img src="https://swift4me.com/images/splash/bigmac2f.png" width="100%">
+<img src="https://swift4me.com/images/splash/bigmac2f.png" width="45%">
```

Comment:
Leave at 100%

Reply:
> Leave at 100%

I have been tested on an Android device with a GitHub mobile app and it seems to be OK
Result: The image is fitting on my Android device excluding on the Web.
Also, I tested on iOS Simulator, and yeah, I think it needs to be changed...
I think 80% may be better.
```suggestion
<img src="https://swift4me.com/images/splash/bigmac1f.png" width="80%">
```